src/Microsoft.Restier.Core/ApiConfiguration.cs

```diff
@@ -60,6 +66,23 @@ public IServiceProvider ServiceProvider
         internal IEdmModel Model { get; private set; }
 
         /// <summary>
+        /// Adds a configuration procedure for API type <typeparamref name="T"/>.
+        /// </summary>
+        /// <typeparam name="T">The API type.</typeparam>
+        /// <param name="configurationCallback">
+        /// An action that will be called during the configuration of <typeparamref name="T"/>.
+        /// </param>
+        [CLSCompliant(false)]
+        public static void Configure<T>(Action<IServiceCollection> configurationCallback)
```

Comment:
Seems this method has no upstream caller and it is not overridable. Is it some kind of new design to facilitate the users?